nixos/maintainers/scripts/ec2/amazon-image-zfs.nix

```diff
@@ -0,0 +1,15 @@
+{
+
+  imports = [ ./amazon-image.nix ];
+  # ec2.efi = true;
```

Comment:
Is there a reason this is commented out? Should either have a comment to document why it's staying, but commented out, or be removed altogether.

(Having seen some of the later stuff, I imagine it's because it's not supported when used with ZFS?)

Reply:
Just cruft from experimenting :)